### DISCUSSION TITLE
Lunar Eclipse Magnitude, Eclipse lists, and Bug fixes

### DISCUSSION THREAD

**Discussion by TheSuggestionGuy** [Stellarium/stellarium#3520] [Ideas]
The umbral magnitude and lunar eclipse obscuration to show up during lunar eclipses on the left side where all the other info is. A list for cities and what they see for an eclipse, like: These cities see a partial solar eclipse, these cities see a Total/Annular Eclipse. This should show up next to the Eclipse tab in "Astronomical Calculations." And please fix the bug where time slows down a bit when you click on the sun, and make the sun's corona also go elliptical, because if sun is super close to horizon while totality, the moon is elliptical and the image of the sun's corona is the same, making it look weird.

**Comment by gzotti** [comment]
Lunar umbral magitudes show up unless you suppress them. Update to 23.3 if you don't see that, I don't remember how long we have it.

You can find the Solar eclipse trail visualisation by downloading and opening a KML file in Google Earth. Stellarium is no general-purpose almanach program which provides data for "A list of cities". There are dedicated eclipse service websites for that purpose: We can be sure the city someone lives in will not be in any list we could devise. How to limit that? Just select your location and check circumstances.

What happens with this bug "where time slows down a bit..."? Please send a clear bug report on that. 

The issue about Solar corona being not distorted by refraction has been reported already. It will be fixed when we find time for it.

**Reply by TheSuggestionGuy** [reply]
So, when you click on the sun, *like when you see the info show up* and if you leave it run for a bit, time will have slowed down, do this to know what I'm talking about: Set time 1 hour back, only one, click on the sun, follow it and have the info that shows up on the side, then let it run for 5 minutes, check the real time and then the Stellarium time, it should be more than an hour back than real time. And for the lunar eclipse magnitude, are you talking about the Magnitude that shows up below the Type? if it is, that's not what I'm talking about

**Reply by alex-w** [reply]
Yes, I can reproduce the issue.

**Reply by gzotti** [reply]
No, I am referring to the eclipse magnitude. 
![grafik](https://github.com/Stellarium/stellarium/assets/18099873/36673c95-a9c6-48a0-9c12-72fc1fdc48e6)

If this is not what you want, please describe what you want and how to compute it.

**Reply by TheSuggestionGuy** [reply]
the obscuration is enough, but I can't find those things, but I've updated to 23.3!

**Reply by TheSuggestionGuy** [reply]
I also can't even find the Libration, Subsolar point, Colongitude, or the Albedo!
But that text is white, my text *I clicked on the moon* is blue?

**Reply by TheSuggestionGuy** [reply]
is this a different object? or is it something else.

**Reply by gzotti** [reply]
You probably have to configure the level of displayed info. User Guide, Fig. 4.4.

**Reply by TheSuggestionGuy** [reply]
where is it? like the setting to turn on and off eclipse magnitude, I did find Show umbra and penumbra shadows in markings.

**Reply by gzotti** [reply]
> User Guide, Fig. 4.4.

**Reply by TheSuggestionGuy** [reply]
I didn't look hard enough, thanks for the lunar eclipse magnitude, disscussion will be closed *sun time error will be in the bug section*

**Comment by TheSuggestionGuy** [comment]
> > User Guide, Fig. 4.4.

**Comment by TheSuggestionGuy** [comment]
Disscussion closed.
